### ISSUE TITLE
Flat slurs

### ISSUE THREAD

**Issue by LupusCaeruleus** [musescore/MuseScore#21782]
### Your idea

Add an alternative style for slurs so that they can have a flat top, as shown in the images below.
![image](https://github.com/musescore/MuseScore/assets/70313425/d2c0c550-dccc-4c95-8a6c-18bd4f24d308)
![image](https://github.com/musescore/MuseScore/assets/70313425/99df6b17-aecc-4ee7-bbfa-4b2592e4ab9b)

### Problem to be solved

When spanning very long phrases, the common arc-shaped slurs creates a lot of vertical space, resulting in an unpleasant appearance and a waste of page space.

To illustrate, here are the first few bars of Ravel's _Gaspard de la Nuit_:
![image](https://github.com/musescore/MuseScore/assets/70313425/437f1158-9daf-4d7f-9b13-2c79c5ba32e9)
![image](https://github.com/musescore/MuseScore/assets/70313425/946957d4-9424-469e-bdc3-4e6209896690)
When recreating the score in MuseScore, extra spaces generated by slurs are quite noticeable:
![image](https://github.com/musescore/MuseScore/assets/70313425/e4689897-867d-4fc4-a723-2bc22420e0a7)

Note that adjusting handles is not helpful because when the height of the slur is reduced, its ends will become too flat.
![image](https://github.com/musescore/MuseScore/assets/70313425/3ce0c780-c5d4-407a-90f4-6b849332ce30)

### Prior art

Lilypond supports custom slur shapes, but actually requires manually specifying every control point of the slur as a Bézier curve. (So it is also possible to create slurs with multiple turning points, but with a lot of effort.)
Dorico has a style setting to "flatten" the slurs.

**Comment by irwir** [comment]
Left-click on a slur line shows grips that could be dragged and thus change the curve.

**Comment by LupusCaeruleus** [comment]
> Left-click on a slur line shows grips that could be dragged and thus change the curve.

Yes and I am aware of this, but adjusting the handles/grips is still not able to yield a satisfactory shape. (See my last screenshot and comment above, note that the top left and top right handles have been pulled as far as they can go on either side, but the ends of the slur are still too flat.)

**Comment by worldwideweary** [comment]
P.S. Fwiw this feature has been asked for ~10+ years such as https://musescore.org/en/node/56446.

**Comment by irwir** [comment]
Thanks for clarifications. 
Apparently, choice between yet another kind of slur or addittional grips is not that great, and this could explain why nobody rushed to implement this feature.
Maybe an alternative should be considered?
If slur line is very long and flat, automatically make ends slightly more curved.

**Comment by yue-na** [comment]
I do like the appearance of the flat slur in SCORE. Looks very beautiful and pleasant.
![830](https://github.com/user-attachments/assets/70799b40-2ec6-48c8-8855-138e8394916f)

**Comment by Dima-S-Jr** [comment]
I would like to add that even on small phrases (about 2 bars), the slur can move significantly away from the staff. Take a look at this, for example. There is an alternation of the 1st and 2nd voices. If the last note of the 2nd voice differs significantly in pitch from the previous ones, the slur will "bend" very much and turn into a huge "hill".

https://github.com/user-attachments/assets/920e65e2-0f7c-4202-8360-93a53f86363b

**Comment by odhot** [comment]
Are there any updates regarding this feature? If I’m not mistaken, MuseScore 2 has more than four handles, cmiiw.

**Comment by MarcSabatella** [comment]
MuseScore 4 slurs have five handles, not four - this is the same as it has always been.
